packages/node_modules/@webex/react-component-md-text-input/src/index.js

```diff
@@ -53,7 +53,7 @@ class MDTextInput extends AdaptiveCard.TextInput {
       <Input
         label={this.title}
         value={this.value}
-        maxLength={this.maxLength}
+        maxLength={this.maxLength || null}
```

Comment:
```suggestion
        maxLength={this.maxLength ?? 0}
```
Are you sure you don't want this? I assume you want to default to '0'

Reply:
@mwegman, I don't think the default value should be `0`, otherwise it will not allow for any input. By using `null` we're basically saying that we don't want to set that property at all

Reply:
@lalli-flores you are correct. @mwegman we have to allow for any input irrespective of `maxLength`

Reply:
So if `this.maxLength` is 0, you want to set it to `null` instead?

Reply:
@mwegman yes, trying to ignore `maxLength` if it is zero